tests/c-chain-tests.js

```diff
@@ -68,10 +68,10 @@ describe("C-chain import and export tests", () => {
     const pathPrefix = "44'/60'/0'";
     const pathSuffixes = ["0/0", "0/1", "100/100"];
     const ui = await flowMultiPrompt(this.speculos, [
-      [{header:"Sign", body:"Import"}],
-      [{header:"Importing",body:"0.268435456 AVAX to local1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqljssag"}],
-      [{header:"Fee", body:"0 AVAX"}],
-      [{header:"Finalize",body:"Transaction"}],
+      {header:"Sign", body:"Import"},
+      {header:"Importing",body:"0.268435456 AVAX to local1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqljssag"},
+      {header:"Fee", body:"0 AVAX"},
+      {header:"Finalize",body:"Transaction"},
```

Comment:
Is there a particular reason we were able to get `Finalize` into the one-array design here, but not in basic-tests? Or is it just a sort of WIP thing

Reply:
Yes this is just what you did. I put my code on top of you commit just changing the tests and merely reduced the number of failures. This *doesn't* work yet.